common/src/Utilities/wxHelpers.cpp

```diff
@@ -95,6 +95,14 @@ void pxFitToDigits( wxWindow* win, int digits )
 	win->SetMinSize( wxSize( ex+10, wxDefaultCoord ) );		// +10 for text control borders/insets and junk.
 }
 
+void pxFitToDigits(wxWindow* win, int digits, int height)
+{
+	int ex;
+	win->GetTextExtent(wxString(L'0', digits + 1), &ex, NULL);
+	win->SetMinSize(wxSize(ex + 10, height));		// +10 for text control borders/insets and junk.
+}
+
```

Comment:
Has this been tested with different DPI settings? I don't think it'll work that well.